### ISSUE TITLE
Why is 'sequence_autoencoder' never used in the next code?

### ISSUE THREAD

**Issue by falhafizh** [cerlymarco/MEDIUM_NoteBook#43]
Hi, Sir
I want to ask, in your notebook [here](https://github.com/cerlymarco/MEDIUM_NoteBook/blob/master/Extreme_Event_Forecasting/Extreme_Event_Forecasting.ipynb)
you define sequence_autoencoder  variabel and train it as TRAIN AUTOENCODER.

`sequence_autoencoder = Model(inputs_ae, out_ae)`
`sequence_autoencoder.compile(optimizer='adam', loss='mse')`

`### TRAIN AUTOENCODER ###`
`es = EarlyStopping(patience=6, verbose=2, min_delta=0.001, monitor='val_loss', mode='auto', restore_best_weights=True)`
`sequence_autoencoder.fit(X_train, X_train, validation_data=(X_train, X_train),`
`                         batch_size=128, epochs=100, verbose=1, callbacks=[es])`

so, why is sequence_autoencoder variabel never used in the next code?

**Comment by cerlymarco** [comment]
Hi, 

because it's a simple autoencoder. we don't use it for prediction and so on. We simply use it to train weights of `encoded_ae`  which is used in the next code cells.

If u support the project, don't forget to leave a star ;-)

All the best
